### PR TITLE
Fix #196: Add the Scala.js version on the front page.

### LINKED ISSUE #196: Show the latest version somewhere on the front page

### PR DESCRIPTION
**Pull request by rockjam**
Not ideal, but at least visible

### CODE REVIEW

**Comment by rockjam** [comment]
Looks this way now.

image

**Comment by sjrd** [comment]
Hum ... This disturbs the visual left/right balance of the title.

I suggest trying either of:

* Putting the version number on a separate line, below "Scala.js" (I would also suggest a bit more opaque, like 0.7 or 0.8)
* Using the same size and opacity as "Scala.js" (so basically, not changing it)

**Comment by rockjam** [comment]
Here are variations:

* Same font, same line, no opacity
image

* Same font, same line, opacity 70% or so
image

* Smaller font, opacity 70%, separate line
image

Which one is more suitable on your opinion?

**Comment by sjrd** [comment]
My favorite is the last one. IMO it's the one with the best balance.

**Comment by rockjam** [comment]
Same for me

**Comment by rockjam** [comment]
Just updated pull request

File: index.html [hunk 1]
```diff
@@ -9,7 +9,9 @@
                 <img width="100" src="{{ '/assets/img/scala-js-logo.svg' | prepend: site.baseurl }}">
 
                 <div>
-                    <h2 style="margin-top: 10px;">Scala.js</h2><h4>A safer way to build robust<br/> front-end web applications!</h4>
+                    <h2 style="margin: 10px 0;">Scala.js</h2>
+		    <h4 class="version">{{ site.versions.scalaJS }}</h4>
+		    <h4>A safer way to build robust<br/> front-end web applications!</h4>
```
Comment (sjrd) [inline index.html]: Make sure not to introduce tabs. We use spaces only.

**Comment by sjrd** [comment]
Also could you use this commit message format, which is what we use:

    Fix #196: Add the Scala.js version on the front page.

That's all.

File: index.html (hunk 1 above)
Comment (rockjam) [inline index.html]: Oops, sorry for that, will fix

**Comment by rockjam** [comment]
All right, no tabs, and proper commit message - done.

**Comment by sjrd** [comment]
LGTM Thanks!

**Pull request merged**